sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -185,6 +183,8 @@ trait ArraySortUtil extends ExpectsInputTypes with CodegenFallback {
 
 object ArraySortUtil {
   type NullOrder = Int
+  // Least: place null element at the end of the array
+  // Greatest: place null element at the beginning of the array
```

Comment:
Can you add `for ascending order` or something?

Reply:
Sure. I will also fix statements.